web/src/modules/i18n/containers/I18nProvider.js

```diff
@@ -2,135 +2,102 @@
 
```

Comment:
I refactored this to a function component to be same as in web

Reply:
This is not exactly the same though, is it?

Reply:
I think it is. The behavior of the app stayed the same in my tests.

Reply:
Yes, functionality, but you wrote to be the same as in native and not to have the same functionality as before :)

Reply:
no it should behave exactly like before in web. The behaviour is still different between web and native